dissect/target/loaders/phobos.py

```diff
@@ -39,10 +51,10 @@ class PhobosLoader(Loader):
     """
 
     @staticmethod
-    def detect(path):
+    def detect(path: Path):
         return path.suffix.lower() == ".eight"
```

Comment:
If we want to go that route it is better to transform it into a kind of `Scrape` loader, as there is nothing really specifically "Phobos" about this loader. It could be used as one of the fallback loaders.

I think that is a good idea, but maybe not in the context of this PR/ticket.